File: JS/Fundamentals/invertValues.js
// Change values in array: pos to neg, and vice versa

// My Code:
function invert(array) {
    let invArr = [];
    for (let num in array ) {
      invArr.push(array[num] * -1);
    }
    return invArr;
}

// Best Practice:
// const invert = array => array.map(num => -num);

File: Python/Fundamentals/invert_values.py
# Change nums in list: positives to negatives and vice versa

# My Code
def invert(lst):
    inv_lst = []
    for num in lst:
        inv_lst.append(-num)
    return inv_lst






# Best Practice

# def invert(lst):
#     return [-x for x in lst]